### ISSUE TITLE
Porting on mono for Linux and Mac OS compatibility

### ISSUE THREAD

**Issue by Ileca** [morkt/GARbro#523]
Great tool you have here, but for a Linux user, it is quite cumbersome to have to run it through Wine all the time.
Seeing that you use .NET Framework, would you consider porting it to Mono which is a cross-platform open source implementation of .NET Framework (in case you don't know)? I might be all wrong about the feasibility of such project as I know nothing about it but that would be great if you could consider making your software Linux (and Mac OS) compatible.

**Comment by Cosetto** [comment]
he hasn't replied to anyone almost 3 years, maybe you should ask this guy https://github.com/crskycode/GARbro/releases

**Comment by Ileca** [comment]
Ooooh, shoot, that's too bad. For some reason, I was under the impression that this repo was thriving. Probably old memory. I should have checked the dates...
As for your suggestion, you can't reach the owner as he didn't open tickets, and I kinda don't see myself bugging him through email. Maybe, who knows?

**Comment by Cosetto** [comment]
Yeah you can, or find him on discord Crsky#6458. He's back to work from his vacation few days ago so you may wait until he's free again

**Comment by VergeDX** [comment]
FYI, Running with wine (requires dotnet461) works perfectly well.
